tests/unit/test_client.py

```diff
@@ -89,7 +89,6 @@ def _make_list_partitons_meta_info(project, dataset_id, table_id, num_rows=0):
 
 
 class TestClient(unittest.TestCase):
-
```

Comment:
Unless the linter is complaining, don't change whitespace this far from your other changes